src/PersonRow/PersonRow.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const PersonRow = ({
+  name,
+  sex,
+  born,
+  died,
+  motherName: mother,
```

Comment:
Each `person` object should contain objects `father` and `mother`. Not only their names. You can map initial array and use method `find`